### DISCUSSION TITLE
Unexpected Calibration Step Change Investigation

### DISCUSSION THREAD

**Discussion by keenanjohnson** [Ribbit-Network/ribbit-network-frog-hardware#232] [General]
A Ribbit frog sensor owner reported that one of their sensors experienced an unexpected step change.

![image](https://github.com/Ribbit-Network/ribbit-network-frog-hardware/assets/2559382/8fe86f1b-f960-472f-99f9-5d830a8af41b).

This change appears to coincide with our update that enabled the ABC algorithm for the SCD30 sensors.

We actually expect the automatic calibration to take effect after 7 days. The step change here seems to be on the upgrade, which is mysterious and warrants some additional investigation.

**Comment by keenanjohnson** [comment]
The update enabling the ABC algorithm: https://github.com/Ribbit-Network/ribbit-network-frog-software/pull/53

**Comment by hansenni** [comment]
Is it possible that the sensor is "storing" data points to use in the ABC Algorithm, so when you enabled it, the sensor already had enough data to shift?
